source/releasenotes/2024-06-04-release-note-multi-select-filter.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Multi-select filter now available for Pantheon release notes
+published_date: "2024-06-04"
+categories: [documentation, user-interface]
+---
+
+We're introducing another enhancement to our release notes: the multi-select filter. Now, you can filter release notes using multiple tags simultaneously, providing a more customized and efficient way to find relevant updates. Additionally, you can clear all filters at once with a single click.
```

Comment:
I believe an extra line below this is needed to add another paragraph.

I wouldn't call out the clear filter button - that's a product spec but not a noteworthy part of this release.